src/msg/model.rs

```diff
@@ -928,92 +930,130 @@ impl fmt::Display for Msgs {
 
 #[cfg(test)]
 mod tests {
-    use crate::config::model::Account;
-    use crate::msg::body::Body;
-    use crate::msg::envelope::Envelope;
-    use crate::msg::model::Msg;
+    use clap::ArgMatches;
+
+    use crate::{
+        config::model::{Account, Config},
+        ctx::Ctx,
+        msg::{body::Body, envelope::Envelope, model::Msg},
+        output::model::Output,
+    };
 
     #[test]
     fn test_new() {
-        // -- Accounts -
-        let account1 = Account::new_with_signature(Some("Soywod"), "clement.douin@posteo.net", None);
-        let account2 = Account::new_with_signature(None, "tornax07@gmail.com", None);
-
-        // -- Creating message --
-        let msg1 = Msg::new(&account1);
-        let msg2 = Msg::new(&account2);
-
-        // -- Expected outputs --
-        let expected_envelope1 = Envelope {
-            from: vec![String::from("Soywod <clement.douin@posteo.net>")],
-            signature: Some(String::from("Account Signature")),
+        let account = Account::new_with_signature(None, "test@mail.com", None);
+        let config = Config {
+            name: String::from("Config Name"),
+            ..Config::default()
+        };
+        let output = Output::default();
+        let arg_matches = ArgMatches::default();
+
+        let ctx = Ctx::new(&config, &account, &output, "INBOX", &arg_matches);
```

Comment:
I agree that there is a bit too much boilerplate (mostly because of the config). I will find a way to simplify it (I was thinking about writing a testing macro).